src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/NewfriendAssistConfig.java

```diff
@@ -33,6 +33,10 @@
 	 * What is the newbie kit composed of.
 	 */
 	private ItemStack[] newbieKit;
+	/**
+	 * Should we give a kit to people who respawn without a bed/anchor?
+	 */
+	private boolean giveToRandomSpawners;
```

Comment:
You're not actually reading this value from the config

Reply:
is this not what line 85 does?

Reply:
I'm either blind or my diff bugged